package.json

```diff
@@ -43,12 +48,18 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.19.0",
     "file-loader": "^3.0.1",
+    "jasmine": "^3.4.0",
     "jest": "^25.4.0",
+    "karma": "^5.0.2",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-jasmine": "^2.0.1",
+    "karma-webpack": "^3.0.5",
```

Comment:
Are you sure you need the karma and jasmine dependencies?  I just swapped them out for jest instead.  Nothing should be using karma anymore, unless I forgot something.

Reply:
Ah, I missed that. Please remove them if no more vulnerabilities with `npm audit`.